cuml/ridge_regression_demo.ipynb

```diff
@@ -98,10 +96,12 @@
    "outputs": [],
    "source": [
     "%%time\n",
-    "X_cudf = gd.DataFrame.from_pandas(X_train)\n",
-    "X_cudf_test = gd.DataFrame.from_pandas(X_test)\n",
+    "X, y = make_regression(n_samples=n_samples, n_features=n_features, random_state=0)\n",
     "\n",
-    "y_cudf = gd.Series(y_train.values[:,0])"
+    "X = cudf.DataFrame.from_gpu_matrix(X)\n",
```

Comment:
It will save time and memory if we create the dataset only once and then convert it to the other form using cudf functions. We can either convert it to a pandas dataframe or to a cudf dtaaframe depending on the function used to create the dataset. (`from cuml import make_regression` or `from sklearn.datasets import make_regression`)

Reply:
Fixed in local.